storybook/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -52,10 +52,8 @@ const defaultTabs = [
   ...(() =>
```

Comment:
Out of scope, but: why is this a function? You can just destructure `tabsContent.map` directly, no?

Reply:
Fixed in https://github.com/Amsterdam/design-system/pull/1275.